os-eap-txnrecovery/run/python2/module.yaml

```diff
@@ -0,0 +1,13 @@
+schema_version: 1
+name: os-eap-txnrecovery.run
+version: 'python2'
+description: Query OpenShift API running with Python 2.
+execute:
+- script: install_as_root.sh
+packages:
+  install:
+    - python-enum34
```

Comment:
This one, I don't think exists in RHEL 8 / UBI, but it appears to have been replaced by a py3 default. See here:

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/8.0_beta_release_notes/index#package_replacements

(see python-enum34)

Reply:
Yep, sorry my mistake, I misread the diff. This is just the python 2 version.

Reply:
yes, the package is needed only for the python2 where the enum is not available as I think from my limited knowledge of python